### ISSUE TITLE
Energy dashboard: Solar production is not displayed when solar forecast is enabled for countries not covered by the service 

### ISSUE THREAD

**Issue by Eonsim** [home-assistant/frontend#9737]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

In the new Energy dashboard, if you live in a country that is not covered by Solar Forecast system, enabling Solar forecasts will result in the Solar Production graph failing to display any data. Instead it will say "no data available may take 2hours to display" even though there is solar production data in the consumption graph.

### Describe the behavior you expected

If Solar Forecasting is enabled, but no forecast data is available (as live in a country outside the supported region, or some other issue) the solar production should be shown. Rather than the message saying "No data it may take 2 hours for data to display".

### Steps to reproduce the issue

1. Set your country to a non-covered one ie New Zealand
2. Setup up Energy dashboard with your Solar.
3. Wait for Solar production data to show on the lower "Solar production" graph.
4. The data will never show as it can't get a forecast from Solar forecast and thus fails to display both the forecast and the solar production data.

### What version of Home Assistant Core has the issue?

2021.8.3

### What was the last working version of Home Assistant Core?

None

### In which browser are you experiencing the issue with?

Current Chrome, Firefox and Edge

### Which operating system are you using to run this browser?

OSX, Windows, Android

**Comment by HACS-bank** [comment]
In the UK, the fault occurs when forecast_solar is unable to run successfully - in my case if I disable prediction, I can see data:

![image](https://user-images.githubusercontent.com/84039996/129180665-6ac5e093-f7ee-48c0-9aa3-777f66cb3bd8.png)

but if I enable Solar Forecast AT FIRST CONFIGURATION
![image](https://user-images.githubusercontent.com/84039996/129180798-aecf8b12-7547-4c19-a8f3-fe0491964295.png)

then the reported issue occurs. Disabling the forecast, and then reenabling it, gives data display, but after restarting the fault recurs.

Integrations shows that Forecast Solar is unable to get data and the error log shows

2021-08-12 11:22:38 WARNING (MainThread) [homeassistant.config_entries] Config entry 'Bankhead' for forecast_solar integration not ready yet: Rate limit for API calls reached. (error None); Retrying in background

and after a restart

2021-08-12 11:24:27 WARNING (MainThread) [homeassistant.config_entries] Config entry 'Bankhead' for forecast_solar integration not ready yet: No data available for location 57.5049, -4.5002 - Please check on https://re.jrc.ec.europa.eu/pvg_tools/en/#DR (error 5); Retrying in background

**Comment by bramkragten** [comment]
This was fixed in dev.

**Comment by HACS-bank** [comment]
I upgraded to DEV and found no change.  

However I think I have spotted the problem: my PV array faces nearly South and following the instructions in Home Assistant I had set the AZIMUTH to 170 degrees (0 = NORTH).

Investigating the logged error and checking https://re.jrc.ec.europa.eu/pvg_tools/en/#DR it seems that the Azimuth should be 0=SOUTH

Changing the Azimuth in Home Assistant to 0 seems to resolve my issue.
![image](https://user-images.githubusercontent.com/84039996/129402266-64d8fb55-faa3-40e7-b1c0-0fe46e182d9c.png)
